packages/@aws-cdk/aws-codebuild/lib/build-spec.ts

```diff
@@ -8,6 +9,15 @@ export abstract class BuildSpec {
     return new ObjectBuildSpec(value);
   }
 
+  /**
+   * Export a YAML from an object
```

Comment:
I feel like this doc could use some work....how about

```suggestion
   * Create a buildspec from an object that will be rendered as YAML in the resulting CloudFormation template.
```